src/components/SuggestedJourneys/SuggestedJourneys.module.scss

```diff
@@ -0,0 +1,93 @@
+@import '../../styles/base.scss';
+
+.journey {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    flex-grow: 1;
+
+    h2 {
+        margin: 0;
+    }
+
+    a {
+        //position: relative;
+        color: rgba(255, 255, 255, 1);
+        text-decoration: none;
+        background-color: rgb(243, 190, 74);
+        font-weight: 250;
+        font-size: 3em;
```

Comment:
I agree that this is probably a little too large. Maybe play with reducing the font size and increasing the padding?